spec/dummy/db/schema.rb

```diff
@@ -27,12 +27,11 @@
 
   add_index "queue_classic_jobs", ["q_name", "id"], name: "idx_qc_on_name_only_unlocked", where: "(locked_at IS NULL)", using: :btree
 
-  create_table "queue_classic_later_jobs", force: true do |t|
-    t.string   "q_name"
-    t.string   "method"
+  create_table "queue_classic_later_jobs", id: false, force: :cascade do |t|
```

Comment:
Yes, updating schema.  Also having that debate with Travis right now :)

On Mon, Jan 26, 2015 at 3:09 PM, Jean-Philippe Boily <
notifications@github.com> wrote:

> In spec/dummy/db/schema.rb
> <https://github.com/QueueClassic/queue_classic_admin/pull/37#discussion_r23560240>
> :
>
> > @@ -27,12 +27,11 @@
> >
> >    add_index "queue_classic_jobs", ["q_name", "id"], name: "idx_qc_on_name_only_unlocked", where: "(locked_at IS NULL)", using: :btree
> >
> > -  create_table "queue_classic_later_jobs", force: true do |t|
> > -    t.string   "q_name"
> > -    t.string   "method"
> > +  create_table "queue_classic_later_jobs", id: false, force: :cascade do |t|
>
> This should not be here anymore, not trace of queue_classic_later_jobs
> should be left I think.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/QueueClassic/queue_classic_admin/pull/37/files#r23560240>
> .
>



-- 
Mitchell Perilstein <mitchell.perilstein@gmail.com>
(484) 876-1444